### ISSUE TITLE
Output descriptor with both receiving and change paths

### ISSUE THREAD

**Issue by louneskmt** [bitcoinerlab/descriptors#31]
Hey! I am trying to use an output descriptor with both the receiving (0) and change (1) paths, as follow: `wpkh([284bbfa8/84h/1h/0h]tpubDCe8CRwGAiQ2r3rdChcB3Fp8v4GVjYJqoyKKwBczxGVNnawaK21p7sWh8oiMRo26wmSEhQqHGPVofX7t9BnGaSZjbED9WwRxMt8UeTjF64S/<0;1>/*)`. It seems that this library does not support this expression `<0;1>`? I get a `Could not parse descriptor` error while it seems to work well with `0` instead of `<0;1>`. Please let me know if I can be of any help implementing this, or the rationale behind not supporting this.

**Comment by louneskmt** [comment]
Oh okay, it is not directly in the specs but seems like an extension. Would be cool to have this anyway though!

**Comment by landabaso** [comment]
I believe this format is specific to Ledger. It's a more compact way to represent internal/external keypairs, making them easier to display on the device. @bigspider may have more insight into this, especially if it has gained wider adoption.

By the way, this library supports Ledger HWW. You can use standard `*` descriptors, and they are automatically converted to the Ledger format for use on Ledger devices.

**Comment by bigspider** [comment]
The multipath syntax is not (yet?) a standard (albeit already quite widely adopted by many software wallets, I think). There's a PR to add it to descriptors: https://github.com/bitcoin/bitcoin/pull/22838

Ledger's script wallet accounts are based on the [wallet policies BIP proposal](https://github.com/bitcoin/bips/pull/1389), also adopted in BitBox02 and Jade, but not yet standardized either.

**Comment by louneskmt** [comment]
> albeit already quite widely adopted by many software wallets, I think

Yeah, I asked this because I was using an output descriptor from Sparrow for my tests and it is using this multipath syntax. 

@landabaso Do you think it would be appropriate to support this syntax in the library, considering its growing adoption?

**Comment by louneskmt** [comment]
@landabaso BIP388 has been merged: https://github.com/bitcoin/bips/blob/master/bip-0388.mediawiki#descriptor-derivation

**Comment by landabaso** [comment]
I'm currently out of office for a couple of weeks... I'll definitely catch up with this in August. Thanks for the notice @louneskmt .

**Comment by louneskmt** [comment]
FYI https://github.com/bitcoin/bitcoin/pull/22838 has been merged as well!
